app/src/main/java/de/danoeh/antennapod/config/DownloadServiceCallbacksImpl.java

```diff
@@ -13,14 +13,15 @@
 import de.danoeh.antennapod.net.download.serviceinterface.DownloadRequest;
 import de.danoeh.antennapod.fragment.CompletedDownloadsFragment;
 import de.danoeh.antennapod.fragment.QueueFragment;
+import de.danoeh.antennapod.ui.appstartintent.MainActivityStarter;
 
 
 public class DownloadServiceCallbacksImpl implements DownloadServiceCallbacks {
 
     @Override
     public PendingIntent getNotificationContentIntent(Context context) {
         Intent intent = new Intent(context, MainActivity.class);
-        intent.putExtra(MainActivity.EXTRA_FRAGMENT_TAG, CompletedDownloadsFragment.TAG);
+        intent.putExtra(MainActivityStarter.EXTRA_FRAGMENT_TAG, CompletedDownloadsFragment.TAG);
```

Comment:
You can now use `MainActivityStarter.withFragmentLoaded()` here to avoid having to handle the specific extras.